extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -294,6 +313,19 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
       detailsText.setAttribute('aria-hidden', !shouldOpen);
     });
   }
+  /**
+   * onclick event that fires when buy now is clicked, has conversion tracking.
+   * @param {?string} conversionURL
+   * @return {boolean}
+   * @private
+   */
+  onClickBuyNow_(conversionURL) {
+    this.analyticsService_.triggerEvent(
+      StoryAnalyticsEvent.SHOPPING_BUY_NOW,
+      null,
+      {'conversionURL': conversionURL}
```

Comment:
Out of curiosity, how does this track conversion (such as buying something on the out-linked page)?

Reply:
I'm currently working on setting this up: For more info on how conversion tracking works, read this:
https://support.google.com/google-ads/answer/7521212

Reply:
There is no standard format to send the conversion URL: This is because every analytics platform has their own "standard", i.e. Google analytics will be structured differently than Facebook Analytics, Twitter Analytics, Adobe Analytics etc.

The only thing we can do is provide the conversion URL. As long as that is clearly indicated in the payload, tooling will be able to customize it for use with the analytics platforms I listed above.

We do not have to set up and g-tag or anything like that for testing as that is specific to Google analytics. We just need to make sure in the devtools network tab that the payload is structured in the way we intend it to be structured  and tooling will take care of the rest.